deps/tools/common.mk

```diff
@@ -29,6 +31,7 @@ CMAKE_COMMON += -DCMAKE_CXX_COMPILER="$(CXX_BASE)"
 ifneq ($(strip $(CMAKE_CXX_ARG)),)
 CMAKE_COMMON += -DCMAKE_CXX_COMPILER_ARG1="$(CMAKE_CXX_ARG)"
 endif
+CMAKE_COMMON += -DCMAKE_LINKER="$$(which $(LD))"
```

Comment:
I was just copying the style used several lines up for `CC`. I'll go with `$(LD)`. Thanks!